src/imports.jl

```diff
@@ -1,10 +1,13 @@
 import Base:
-  sqrt
+  sqrt,
+  push!
 
 import ITensors:
   # circuits/gates.jl
   space,
-  state
+  state,
+  measurements,
+  norm
```

Comment:
`norm` is originally in `LinearAlgebra`, probably better to import it from there (not sure if it matters).